LeanCV: add theme

henriqueiamarino opened the pull request:
LeanCV is a blog theme that echoes the structure of a professional profile with original visuals and interesting navigation. It's suitable for the general public to display information, experiences, and education. And it's super easy to customize.

iamarinoh committed:
LeanCV: add theme

jasmussen commented:
Theme looks good! I noticed a scroll jump for the 100vh sticky image:
 
![scroll](https://github.com/Automattic/themes/assets/1204802/3c7a500f-a277-46f5-abe5-745ad6d15972)

But this appears _only_ when the adminbar is present. So that's okay, you the sticky is actually a good solution to this.

---

A couple of visual bits of feedback. The block gap in this stack looks a little tight: 
![Screenshot 2024-06-25 at 09 57 49](https://github.com/Automattic/themes/assets/1204802/bcdc3779-1e49-4808-8462-621478e4a622)

It's not quite big enough to look intentional, and bumping it just a bit would help. The spacing feedback applies to some of the rest of the page too:
![Screenshot 2024-06-25 at 09 57 58](https://github.com/Automattic/themes/assets/1204802/df76fdac-0609-4eed-ae5b-dca8a331bca1)

That may be a personal opinion, but just bumping the gaps a little bit might aid legibility. Here's a quick test:
![Screenshot 2024-06-25 at 10 02 09](https://github.com/Automattic/themes/assets/1204802/9c2a0631-f948-4f40-8a7d-92c0d74b01ac)

Not strong opinions, let me know if it resonates.

---

Templates look good. I think across all the theme work, it might be good to do a separate exercise to improve our "no search results found" text. While it can be different from theme to theme, in most cases IMO it can be the same. And these two read a little flowery to me:
![Screenshot 2024-06-25 at 10 05 16](https://github.com/Automattic/themes/assets/1204802/03b58743-6049-4789-a71c-dd4c5ad261a7)

![Screenshot 2024-06-25 at 10 05 47](https://github.com/Automattic/themes/assets/1204802/75dca295-7ab2-4bae-aa5a-b9cc2702cb0f)

Something simpler like "No results found." and "You can try a search:" might be better for both cases. I don't think you necessarily have to address that for this theme, but it's something to generally consider across themes. CC: @beafialho.

---

Style variations look good. Blue is a good professional default, but the red wine and forest green look great too. Can we find a different name than vampirella?
![Screenshot 2024-06-25 at 10 06 13](https://github.com/Automattic/themes/assets/1204802/f575542b-c797-45e8-b0c7-173bd2582e33)

Vampirella is a trademark! 😅 

---

In a quick code review, I found this lovely picture:

![Screenshot 2024-06-25 at 10 04 44](https://github.com/Automattic/themes/assets/1204802/3d1d17b2-5c54-4f37-b68e-dd0bd54dd5f5)

Is that used anywhere? Should it be? I assume the featured person is generated using Midjourney. Same feedback as for "Outnow" for the readme, would be good to change 

> All images used in the templates were generated with AI using Midjourney.

to

> Generated with AI on Midjourney, released under the CC0 license

So in summary, looks good, nice CV theme. Let's get the style variation renamed, the README updated, check if the mary shelley picture needs to be there, and consider whether to do any spacing changes or not, and then this one is good to go. Nice work!

iamarinoh committed:
Addressing design feedback from Joen

henriqueiamarino commented:
Thanks, @jasmussen. The demo site is live [here](https://leancv.mystagingwebsite.com/). I conducted the following changes:
- Removed the Mary Shelley portrait from the assets. It was the V1 launched in Dotcom, but Ian required a contemporary photography to make the theme more appealing;
- Made spacing more abundant;
- Fixed no-results text;
- Renamed style variations.

henriqueiamarino commented:
@jasmussen I closed this PR as I can't revert a commit. I created a new one: #7893
